index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Register</title>
+  </head>
+  
+
+  <body>
+    <header>Assignment 1</header>
+    <link href="./CSS/style.css" rel="stylesheet" />
+    <h1>Register</h1>
+    <form id="form">
+     
+        <div class="formfield">
+          <label for="first"> First Name:</label>
+          <input type="text" name="firstName" id="firstname" class="input" required/>
+        </div>
+        <div class="formfield">
+          <label for="last"> Last Name:</label
+          ><input type="text" name="lastname" id="lastname" class="input" />
+        </div>
+        
+        <div class="formfield"><label for="email">Email:</label>
+          <input type="email" name="email" id="email" class="input" />
+        </div>
+        <div class="formfield"><label for="gender">Gender:</label>
+          <input type="radio" id="male" name="Gender" value="male" />
+          <label for="male">Male</label>
+          <input type="radio" id="female" name="Gender" value="female" />
+          <label for="female">Female</label>
+          <input type="radio" id="other" name="Gender" value="other" /><label
+            for="other">Other</label>
+        </div>
+        <div class="formfield"> <label for="favourite" class="input">Favourite Color:</label
+          ><select id="favclr" name="favclr">
+          <option value="" selected></option>
+          <option value="yellow">Yellow</option>
+          <option value="red">Red</option>
+          <option value="green">Green</option>
+          <option value="purple">Purple</option>
+          </select>
+        </div>
+
+        <div class="formfield">
+          <label>Employed</label>
+          <input type="checkbox" id="check" class="input" value="check" name="employed"/>
+        </div>
+        <div class="formfield">
+          <label for="notes">Notes:</label>
+          <textarea id="notes" name="notes" rows="2" cols="8" class="input"></textarea>
+        </div>
+        <button type="submit" value="save" >Save</button>
+        <button>Clear</button>
+    </form>
+    <div id="formData">
+
+    </div>
+    
+  <div>
+    <h1>List</h1>
+    <div class="List">
```

Comment:
classnames should be all smallcase or camelCase, should not begin with capital letter